src/doc_scan_service/session/retrieve/third.party.identity.fraud.1.check.response.js

```diff
@@ -0,0 +1,21 @@
+const GeneratedProfileResponse = require('./generated.profile.response');
+const CheckResponse = require('./check.response');
+
+class ThirdPartyIdentityFraud1CheckResponse extends CheckResponse {
```

Comment:
Please have a look at `ThirdPartyIdentityCheckResponse` as it is doing the same thing, except it actually gets that for free as it extends the `ProfileCheckResponse`... so we should do the same for this one.